source/_components/switch.telnet.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Telnet Switch"
+description: "Instructions how to integrate telnet switches into Home Assistant."
+date: 2017-08-10 19:19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Switch
+ha_release: 0.51
```

Comment:
Next release will be 0.51 but this most likely won't be able to merged into that since 0.51 is rather imminent. 
Suggest you update this to 0.52.